pkg/provider/userinterface.go

```diff
@@ -0,0 +1,12 @@
+package provider
+
+// User is a struct that provides json information
+type User struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+// Handler is an interface
```

Comment:
Handler is an interface? Is that your comment? Interface for what? `provider.Handler`? what does that mean?